tests/NHSD.BuyingCatalogue.API.IntegrationTests/Steps/Entities/SolutionEntitySteps.cs

```diff
@@ -107,6 +116,79 @@ public static async Task LastUpdatedHasUpdatedOnMarketingContact(string solution
             (await contact.LastUpdated.SecondsFromNow()).Should().BeLessOrEqualTo(5);
         }
 
+
+        [Given(@"SolutionDetail exist")]
+        public static async Task GivenSolutionDetailExist(Table table)
+        {
+            foreach (var solutionDetail in table.CreateSet<SolutionTable>())
+            {
+                //await CatalogueItemEntityBuilder.Create()
+                //    .WithCatalogueItemId(solutionDetail.SolutionId)
+                //    .WithName(solutionDetail.SolutionName ?? "SomeName")
+                //    .WithSupplierId(solutionDetail.SupplierId ?? "Sup 1")
+                //    .WithPublishedStatusId((int)solutionDetail.PublishedStatus)
+                //    .Build()
+                //    .InsertAsync();
```

Comment:
Why is this commented out? An entry in the CatalogueItem table is required.

Reply:
Catalogue Item are inserted as part of Solution Entry and not Solution Detail update. 
Commented lines are removed.